.github/workflows/ci.yml

```diff
@@ -194,3 +194,20 @@ jobs:
     - name: Check formatting
       run: |
         cargo fmt --all -- --check
+
+  docs:
+    name: Docs
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+      - name: Check documentation
+        env:
+          RUSTDOCFLAGS: -D warnings
+        run: cargo doc --no-deps --document-private-items
```

Comment:
This should probably have `--all` in it, right?

Reply:
Yes, I also added `--all-features`